sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -113,6 +126,22 @@ def __init__(self, type_uri=CUSTOM_REFERENCE_NS, identity="customReferenceProper
             self.location = sbol3.OwnedObject(self, f"{self.CUSTOM_REFERENCE_NS}#location", 0, math.inf, type_constraint=sbol3.Range)
 
 
+    class CustomStructuredCommentProperty(sbol3.CustomTopLevel):
+        """Serves to store information and annotations for 'Structured_Comment' objects in 
+        GenBank file to SBOL3 while parsing so that it may be retrieved back in a round trip
+        :extends: sbol3.CustomTopLevel class
+        """
+        CUSTOM_STRUCTURED_COMMENT_NS = "http://www.ncbi.nlm.nih.gov/genbank#structured_comment"
+        def __init__(self, type_uri=CUSTOM_STRUCTURED_COMMENT_NS, identity="customStructuredCommentProperty"):
```

Comment:
I think this should use the defined constant `CUSTOM_STRUCTURED_COMMENT_PROPERTY_URI` instead of defining a new constant. But maybe the two represent different things and it is coincidental that their values are the same. Use your judgment here, I may be incorrect about this.